specification/search/data-plane/Microsoft.Azure.Search.Service/preview/2017-11-11-preview/searchservice.json

```diff
@@ -4475,6 +4517,125 @@
 		},
 		"description": "Response from a List skillset request. If successful, it includes the full definitions of all skillsets."
 	},
+	"TextExtractionAlgorithm": {
+		"type": "string",
+        "enum": [
+          "printed",
+          "handwritten"
+        ],
+        "x-ms-enum": {
+          "name": "TextExtractionAlgorithm",
+          "modelAsString": false
+        },
+		"description": "A value indicating which algorithm to use. Default is printed."
+	},
+	"TextSplitMode": {
+		"type": "string",
+        "enum": [
+          "pages",
+          "sentences"
+        ],
+        "x-ms-enum": {
+          "name": "TextSplitMode",
+          "modelAsString": false
+        },
+		"description": "A value indicating which split mode to perform. The supported values are 'pages' and 'sentences'."
```

Comment:
I believe AutoRest generates the list of values for you when it generates comments in the SDK code. You might want to double-check that and ensure these values don't appear twice in the generated code.

Reply:
Maybe the enum values are enough. Removed the second sentence. Thanks.